src/libtorch.cc

```diff
@@ -194,6 +204,20 @@ ModelState::LoadModel(
             "' for model instance '" + Name() + "'");
   }
 
+  // Skip loading model if it is already available on the target device
+  auto device_pair = std::make_pair(!device.is_cpu(), device.index());
+  auto mit = torch_models_.find(device_pair);
+  if (mit != torch_models_.end()) {
+    *torch_model = mit->second;
+    LOG_MESSAGE(
+        TRITONSERVER_LOG_INFO,
+        (std::string("Re-using model for pytorch for model instance '") +
```

Comment:
Remove the "for pytorch" and reword to "Re-using TorchScript model for instance ..."